src/cephadm/cephadm

```diff
@@ -2338,6 +2338,18 @@ def command_rm_daemon():
          verbose_on_failure=False)
     call(['systemctl', 'disable', unit_name],
          verbose_on_failure=False)
+    if daemon_type == 'osd':
+        CephContainer(
+            image=args.image,
+            entrypoint='/usr/sbin/ceph-volume',
+            args=[
+                'lvm', 'zap', '--osd-id',
+                str(daemon_id)
+                # not sure if --destroy is useful here
```

Comment:
Hm. I think it's the job of mgr/cephadm to do that, right?

Reply:
Not sure it is. Conceptually it belongs to a `rm_daemon` operation, right? It's already running on the machine and has all the data it needs.
If we call if from mgr/cephadm, we'd have to make the call remotely ontop of the existing call to `rm_daemon`.

The upside of moving it  to mgr/cephadm is that we can control if we want to zap the devices or not. This is, at least not trivially doable with the cephadm approach.